includes/integrations/class-partial-entries.php

```diff
@@ -306,6 +308,15 @@ public function filter_gform_pre_render( $form ) {
 		return $form;
 	}
 
+	/**
+	 * Returns a message to indicate that the Partial Entry was completed.
+	 *
+	 * @since 2.5
+	 */
+	public function partial_entry_complete() {
+		return '<p>' . esc_html__( 'This entry has completed.', 'gravityforms' ) . '</p>';
```

Comment:
The text domain would be gravityflow:
```suggestion
		return '<p>' . esc_html__( 'This entry has completed.', 'gravityflow' ) . '</p>';
```